backend/config/settings.py

```diff
@@ -83,6 +83,11 @@
     },
 }
 
+logging.getLogger('boto3').setLevel(logging.CRITICAL)
+logging.getLogger('botocore').setLevel(logging.CRITICAL)
+logging.getLogger('nose').setLevel(logging.CRITICAL)
+logging.getLogger('s3transfer').setLevel(logging.CRITICAL)
```

Comment:
this is an unrelated change that I can remove if desired. By default, `boto3` (and related libs) are configured with a `DEBUG` log level, which is very noisy and not particularly useful.